scss/_components.scss

```diff
@@ -64,10 +56,6 @@ label {
   cursor: pointer;
 }
 
-.content p {
-  color: #666;
```

Comment:
Hmm, do you think the contrast is too low without this, for readability?

![screen shot 2016-11-18 at 12 12 47 pm](https://cloud.githubusercontent.com/assets/82317/20441046/59a145ce-ad88-11e6-8450-13e1ae5973b7.png)